src/traces/mesh3d/defaults.js

```diff
@@ -86,4 +86,7 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
     }
 
     coerce('text');
+
+    // disable 1D transforms
+    traceOut._length = null;
```

Comment:
Actually, `mesh3d` only expects 1D data, so `_length` could have a meaning and transforms could work.

Reply:
Ah nevermind, i/j/k and x/y/z lengths don't match for `mesh3d` traces. Can you add a comment about this above this line like you did for `sankey` traces?